resources/contributors.txt

```diff
@@ -42,6 +42,7 @@ Iulian Onofrei | https://github.com/revolter | :/avatars/revolter.jpg | Contribu
 matthewde | https://github.com/m4tthewde | :/avatars/matthewde.jpg | Contributor
 Karar Al-Remahy | https://github.com/KararTY | :/avatars/kararty.png | Contributor
 Talen | https://github.com/talneoran | | Contributor
+SLCH | https://github.com/SLCH | | Contributor
```

Comment:
Isn't this meant to be in a separate pr?

Reply:
i have no idea, could reverse it.

Reply:
Reverted

Reply:
> i have no idea, could reverse it.

Better to be a separate PR incase reverting of this PR happens, so your Contributor entry isn't lost.

Reply:
It can be done in the same PR - I did this myself back in the old days. Feel free to include it in this one, I think there's no harm but if you wanna farm some more commits do it separetly.

Reply:
> incase reverting of this PR happens

Why would it though? :thinking: